// func/dist/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Container = exports.CommandMissing = exports.CommandNotFound = exports.CommandMajor = exports.Command = exports.SubOptions = exports.Option = void 0;
require("reflect-metadata");
var annotations_1 = require("./annotations");
Object.defineProperty(exports, "Option", { enumerable: true, get: function () { return annotations_1.Option; } });
Object.defineProperty(exports, "SubOptions", { enumerable: true, get: function () { return annotations_1.SubOptions; } });
Object.defineProperty(exports, "Command", { enumerable: true, get: function () { return annotations_1.Command; } });
Object.defineProperty(exports, "CommandMajor", { enumerable: true, get: function () { return annotations_1.CommandMajor; } });
Object.defineProperty(exports, "CommandNotFound", { enumerable: true, get: function () { return annotations_1.CommandNotFound; } });
Object.defineProperty(exports, "CommandMissing", { enumerable: true, get: function () { return annotations_1.CommandMissing; } });
var container_1 = require("./containers/container");
Object.defineProperty(exports, "Container", { enumerable: true, get: function () { return container_1.Container; } });
__exportStar(require("./services"), exports);
